components/Manual.tsx

```diff
@@ -54,6 +54,13 @@ function Manual() {
   useEffect(() => {
     getTableOfContents(version ?? "master")
       .then(setTableOfContents)
+      .then(() =>
+        setTimeout(
```

Comment:
Maybe `requestAnimationFrame` instead of `setTimeout` 0?

Reply:
`setTimeout` is used in other places too:

https://github.com/denoland/deno_website2/blob/4506a164821cf4f222bb65fd8d1e92fe36ed7a8a/components/Markdown.tsx#L110-L119

And `requestAnimationFrame` is not a **macrotask** like `setTimeout`. It may not trigger on every event loop.